parcels/particle.py

```diff
@@ -257,7 +314,8 @@ def remove(self, indices):
         return particles
 
     def execute(self, pyfunc=AdvectionRK4, starttime=None, endtime=None, dt=1.,
-                runtime=None, output_file=None, output_interval=-1, show_movie=False):
+                runtime=None, interval=None, output_file=None, tol=None,
+                show_movie=False):
```

Comment:
Since complex multi-level sub-timestepping will require `interval` to be derived from multiple sources, I currently favour the more general term. I understand that this is not the most user friendly, but this variable should not actually be set by users once we have a more detailed API for our increasingly complex time loop. In the meantime I believe a detailed docstring should do the trick, while we should really focus on designing a clean and concise time stepping API.